web/yo/app/views/gene.html

```diff
@@ -365,13 +361,13 @@ <h3 ng-if="reviewMode">Gene Type:
                                  <e-status ng-click="commentClick($event)" ng-keypress="stopCollopse($event)" object="treatment.name_eStatus" id="mutation.name.text + tumor.name.text + ti.name.text + treatment.text" apply-obsolete="applyObsolete(treatment.name_eStatus, 'treatment', mutation, tumor, ti, treatment)"></e-status>
                                 <i ng-click="remove($event, 'treatment', mutation, tumor, ti, treatment)" class="fa fa-trash-o"></i>
                               </span>
-                              <div ng-if="treatment.name_review.get('review') !== 'TREATMENT_NAME_CHANGE'">
+                              <div ng-if="!treNameChangeOnly">
```

Comment:
We have displayCheck function for uib-accordion-group to decide if this section need to be displayed or not, treNameChangeOnly only works when there are changes happened in the treatment. But there is a problem rising up here, treNameChangeOnly will be applied to all treatment that has any change. I am working on it.